doc/samples/sample_notify_fifo.sh

```diff
@@ -47,6 +62,7 @@ start_shutdown()
 	) &
 }
 
```

Comment:
Add:
```
reload_terminate()
{
    exit 0
}
```

and
`trap reload_terminate QUIT`

so the script knows why it is being killed and can cleanup appropriately. This also means that the `flock -u` is executed.